src/lib/DatePicker.svelte

```diff
@@ -453,4 +510,42 @@
       color: var(--date-picker-selected-color, inherit)
       background: var(--date-picker-selected-background, rgba(2, 105, 247, 0.2))
       border: 2px solid var(--date-picker-highlight-border, #0269f7)
+  .timepicker
```

Comment:
Would be nice for this to more closely match the other input/select styles

Reply:
Just to confirm, by other input/select are you talking about the styles for the month and year selector?

Reply:
Mainly the `DateInput` textfield, but also the month/year `<select>` elements yeah